shared/js/background/background.js

```diff
@@ -25,3 +28,16 @@ require('./script-injection')
 settings.ready().then(() => {
     onStartup()
 })
+
+/**
+ * @type {{
+ *  fireButton?: FireButton;
+ * }}
+ */
+const features = {}
+
+if (getBrowserName() !== 'moz') {
+    features.fireButton = new FireButton({ settings, tabManager })
+}
+console.log('Loaded features:', features)
+self.features = features
```

Comment:
How come we assign it to `self.features`?

Reply:
This is for debugging (so the class instance can be accessed in the extension debug console), and for integration tests, where we call the class functions.